sugarscape.py

```diff
@@ -940,12 +940,17 @@ def printHelp():
     exit(0)
 
 def verifyConfiguration(configuration):
+    for configName, configValue in configuration.items():
+        if isinstance(configValue, list):
+            if configName != "environmentPollutionDiffusionTimeFrame" or configName != "environmentPollutionTimeFrame":
```

Comment:
I think this `or` should be an `and`. Otherwise, one conditional will be true for each, and they will still be sorted.